### ISSUE TITLE
Listing request - Battle Token

### ISSUE THREAD

**Issue by KingXerxes** [forkdelta/tokenbase#416]
Contract address:
https://etherscan.io/address/0x2accaB9cb7a48c3E82286F0b2f8798D201F4eC3f

Name: BATTLE
Full Name: Battle Token
Decimals: 18

Please list the token with full name BATTLE 

Issuer's official website:
http://persians.network/

Official contract address confirmation:
http://persians.network/
https://github.com/Neurone/persians/
https://etherscan.io/token/0x2accaB9cb7a48c3E82286F0b2f8798D201F4eC3f

A link to a third-party review or discussion of the token:
https://bitcointalk.org/index.php?topic=2592369.0
https://t.me/BattleToken

Description:
Our team launches a platform/ecosystem for gaming, built on top of decentralized smart contracts, allowing anyone to use a Warrior tokens: entities which exist on the Ethereum Network, which can be traded or used to fight in battle with other Warrior tokens holders 
The fuel of the platform is the Battle Token, which can be used to interact with every function such as buy warrior token and special abilities or unlock special events and special battle smart contracts.

**Comment by Kryptoniq** [comment]
Hi, the BTL ticker is aready in use by another project.

**Comment by KingXerxes** [comment]
It is possibile to list with the name BATTLE ?

**Comment by Kryptoniq** [comment]
@KingXerxes yes it is possible. I'll get back to this issue.

**Comment by KingXerxes** [comment]
@Kryptoniq Thank you , I have modified the request :)

**Comment by KingXerxes** [comment]
@Kryptoniq have some updates ?

**Comment by Kryptoniq** [comment]
Custom symbol: BATTLE 
Thanks for your request, we have verified that the information provided is accurate. We'll be listing the token in the next few days, you'll receive an update when we do.
0x2accab9cb7a48c3e82286f0b2f8798d201f4ec3f, http://persians.network/

**Comment by KingXerxes** [comment]
Thanks!

**Comment by freeatnet** [comment]
@KingXerxes There seem to be little activity around the project and the "roadmap" on http://persians.network/ ends at 31 October 2017. Can you elaborate on what the future for the project holds?

**Comment by freeatnet** [comment]
@KingXerxes @Kryptoniq I'm not quite sure who would trade these tokens or what this project plans going forward. I'd like to understand what value they bring to ForkDelta users.

**Comment by KingXerxes** [comment]
Hi @freeatnet , the token is the basis of a project on which we are working on from one year.
We are a small team, we started work on this project starting on the idea that token could be use for fun.
We started generating two contracts that were used to fight on a smart contract (we provided for DAPP for battle monitoring), despite our possibilities the project had a fair success.
After the first battle contract we had decided to work an a platform for gaming on eth blockchain. To fueling the platform we had make the Battle Token and, to reach as many people as possible, we decide to make an airdrop splitted in many phase. The first two phase as ended and at the moment nearly 500 token holder had decided to jump (for free) in our platform.
We are working on the project, we strongly want our project to be started but there are many work to do and we do not want to release a slow or untested platform. The success of our platform is linked to the diffusion of knowledge of the project, so we had decided to submit the request for listing Battle Token on your system. This way the holders or investors who want to bet on the project will help increasing interest on the token.
If you want more specific info don't hesitate to let us know. Many thanks.
